src/main/java/no/uib/inf112/core/RoboRally.java

```diff
@@ -28,7 +28,9 @@ public RoboRally(String mapPath, int playerCount) {
         playerHandler = new PlayerHandler(playerCount, map);
         for (IPlayer player : playerHandler.getPlayers()) {
             map.addEntity(player);
+            player.update(true);
```

Comment:
There is no need for an update here. It will be automatically created.

Reply:
the robots doesn't appear in the start without this, but it could possibly be moved to the addEntity method

Reply:
👍